src/ctelnet.cpp

```diff
@@ -4,7 +4,8 @@
  *   Copyright (C) 2013-2014 by Stephen Lyons - slysven@virginmedia.com    *
  *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
  *   Copyright (C) 2015 by Florian Scheel - keneanung@googlemail.com       *
- *   Copyright (C) 2016 by Ian Adkins - ieadkins@gmail.com                 *
+ *   Copyright (C) 2017 by Darksix - https://launchpad.net/~deathlire      *
+ *   Copyright (C) 2017 by RockHound                                       *
```

Comment:
I think you have to use a real name - a pseudonym does not necessarily identify the legal entity that is claiming the copyright - and the absence of a clear statement of who has contributed to the project would make subsequent tracing such entities difficult!